lib/checkersreport.cpp

```diff
@@ -154,8 +155,7 @@ std::string CheckersReport::getReport(const std::string& criticalErrors) const
     int maxCheckerSize = 0;
     for (const auto& checkReq: checkers::allCheckers) {
         const std::string& checker = checkReq.first;
-        if (checker.size() > maxCheckerSize)
-            maxCheckerSize = checker.size();
+        maxCheckerSize = std::max<std::basic_string<char>::size_type>(checker.size(), maxCheckerSize);
```

Comment:
`maxCheckerSize` can be `size_t`. This could also be a `useStlAlgo` FN.

Reply:
Done.